### ISSUE TITLE
command to create a wallet fails

### ISSUE THREAD

**Issue by vrde** [C-ORG/c-org#4]
I've followed the README and copied the file `config.yaml` in my `~/.c-org/cocorico` project. Running `c-org wallet create cocorico` on my terminal gives me the folliwing error:
```
➜  c-org git:(master) c-org wallet create cocorico
Traceback (most recent call last):
  File "/home/vrde/.local/share/virtualenvs/c-org/bin/c-org", line 11, in <module>
    load_entry_point('c-org', 'console_scripts', 'c-org')()
  File "/home/vrde/repos/c-org/c-org/c_org/cli/__init__.py", line 21, in main
    c_org.main()
  File "/home/vrde/repos/c-org/c-org/c_org/cli/core.py", line 46, in main
    self.run_command()
  File "/home/vrde/repos/c-org/c-org/c_org/cli/command.py", line 116, in run_command
    self.func()
  File "/home/vrde/repos/c-org/c-org/c_org/cli/commands/wallet.py", line 75, in run
    self.run_command()
  File "/home/vrde/repos/c-org/c-org/c_org/cli/command.py", line 116, in run_command
    self.func()
  File "/home/vrde/repos/c-org/c-org/c_org/cli/commands/wallet.py", line 95, in command_create_wallet
    if vault.exist_wallet(name=self.name):
  File "/home/vrde/repos/c-org/c-org/c_org/manager/vault.py", line 128, in exist_wallet
    return self.exists(name, 'name', 'wallets')
  File "/home/vrde/repos/c-org/c-org/c_org/manager/base.py", line 120, in exists
    data = self.get(*keys)
  File "/home/vrde/repos/c-org/c-org/c_org/manager/base.py", line 94, in get
    data = self.data
  File "/home/vrde/repos/c-org/c-org/c_org/manager/base.py", line 46, in data
    self.load()
  File "/home/vrde/repos/c-org/c-org/c_org/manager/base.py", line 76, in load
    with open(self.filename, 'r') as f:
FileNotFoundError: [Errno 2] No such file or directory: '/home/vrde/.c-org/vault.yaml'
```

I've noticed that in the tests there are some extra configuration files, namely `vault.yaml` and `global.yaml`. If I copy both `./tests/ressources/.c-org/global.yaml` and `./tests/ressources/.c-org/vault.yaml` in `~/.c-org/` the command runs successfully.

**Comment by guhur** [comment]
Ok I have a look at it this morning. `vault.yaml` is a temporary solution that is why it should not appear in the README

**Comment by guhur** [comment]
The files `vault.yaml` and `global.yaml` were supposed to be created during the setup of the package.
How did you install the utility?

**Comment by vrde** [comment]
>  How did you install the utility?

I used `pip install -e .`, but at one point I removed `~/.c-org` because I wanted to start from scratch. I've run `pip install -e .` again and now see the files.

IMO the `~/.c-org` directory and the two `yaml` files should be created by the CLI when they are missing, not on the installation of the package. Or they can be generated by `c-org init`. This is the kind of behavior most commands/applications have: they create their own home directory when missing. What do you think?

**Comment by guhur** [comment]
Makes sense. This is fixed on commit 85602ae
